updater/fileHandling_test.go

```diff
@@ -0,0 +1,70 @@
+package updater
+
+import (
+	"github.com/Flaque/filet"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestLocalClientReadData(t *testing.T) {
+	//arrange
+	defer filet.CleanUp(t)
+	CdnBaseUrl, _ := os.Getwd()
+	testFile := "go-testLocalClientRead.txt"
```

Comment:
Usage of  `fs` and `//embed` ?
https://benjamincongdon.me/blog/2021/01/21/A-Tour-of-Go-116s-iofs-package/